packages/bridge/src/squid/transfer-status.ts

```diff
@@ -15,14 +15,13 @@ import { squidProviderId } from ".";
 export class SquidTransferStatusProvider implements TransferStatusProvider {
   readonly keyPrefix = squidProviderId;
   readonly sourceDisplayName = "Squid Bridge";
-
   public statusReceiverDelegate?: TransferStatusReceiver;
 
-  readonly apiUrl: string;
+  readonly apiURL: string;
   readonly squidScanBaseUrl: string;
 
   constructor(env: BridgeEnvironment) {
-    this.apiUrl =
+    this.apiURL =
```

Comment:
Super nit: but I thought with acronyms convention was only to capitalize the first letter?

Reply:
I think this was a regression. I'll push a fix